qa/config/generate_test_data.pl

```diff
@@ -537,7 +537,7 @@ BEGIN
 
       Bugzilla::User->create({
         login_name    => $watch_user,
-        cryptpassword => Bugzilla->passwdqc->generate_password(),
+        cryptpassword => generate_random_password(14), # Increase to pass password complexity
```

Comment:
This comment doesn't make sense

Reply:
I will make it more descriptive of what is going on. Basically the way generate_random_password was working, I was getting validation errors since sometimes it would not include enough of the qualities to pass. Making the password 14 characters long instead of the default 12, made it then pass each time when creating a new user account.